.storybook/stories/templates/Login-page.stories.mdx

```diff
@@ -29,7 +29,12 @@ export const navigationPathData = [
 ];
 
 <Canvas>
-  <Story name="Login page">
+  <Story
+    name="Login page"
+    parameters={{
```

Comment:
Ah I would have thought these parameters to be part of the `<Meta>` so all login stories would be tested for various widths. Do you think it's enough to have this single story for mobile?